### PR TITLE
Bug 1779797 - part 2-6: Let mergify automatically merge l10n bumps to main

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Ports https://github.com/mozilla-mobile/fenix/pull/26073 to Focus. Fixes up https://github.com/mozilla-mobile/focus-android/pull/7369

Tested with a bump on:
 * `main`: https://github.com/mozilla-releng/staging-focus-android/pull/6
 * a release branch: https://github.com/mozilla-releng/staging-focus-android/pull/7

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -11,7 +11,15 @@ treeherder:
...
         'TL': 'Toolchain builds for Linux 64-bits'
 
-task-priority: highest
+task-priority:
+    by-project:
+        "focus-android": highest
+        "staging-focus-android": low
+        # This handles cases where a fork may end up not being named
+        # as one of the explicit names above. Ideally anything forked
+        # from "fenix" would inherit its setting, but "low" is the
```
Comment (ahal) [inline taskcluster/ci/config.yml:20]: nit: `s/fenix/focus-android`

**Review by ahal — APPROVED** [review]

File: taskcluster/ci/config.yml (hunk 1 above)
Comment (JohanLorenzo) [reply taskcluster/ci/config.yml]: Oops, fixed!

**Comment by JohanLorenzo** [comment]
@mergifyio update

**Comment by JohanLorenzo** [comment]
Oh! Mergify can't merge this patch for me on this repo[1]:

> ⚠️ The configuration has been changed, queue and merge actions are ignored. ⚠️

It was no issue on fenix, though. @mozilla-mobile/focus-android-eng, can someone manually merge this PR for me? I guess it's not worth investigate this very difference between focus and fenix since we may merge these repo all together soon.

[1] https://github.com/mozilla-mobile/focus-android/pull/7397/checks?check_run_id=7446866364

**Review by Mugurell — APPROVED** [review]

**Pull request merged**